### ISSUE TITLE
Go vs Rust

### ISSUE THREAD

**Issue by saadjameel** [apache/age-viewer#97]
For the red development of AGE viewer, I have deeply studied Go and Rust. 
I found Rust more convenient than Go due to it's stability and more libraries.

Everyone please suggest with a little description which one to be chosen.

**Comment by marodins** [comment]
I like the idea of using Go with Echo and Wails. There's good support and documentation for these frameworks.

**Comment by MJinH** [comment]
I agree with the idea of using golang for this project. It has widely used frameworks and well-written documents. I prefer gin and wails tho.

**Comment by Nimra-1234** [comment]
I also agree with GO due to vast documentation and have strong support.

**Comment by AhmarZaidi** [comment]
I also believe Go is the way to go 😄.
Go has a larger and more mature ecosystem than Rust. Wails had good documentation as well.

**Comment by Pratikkapsime** [comment]
I also prefer that GO is suitable for a large team as it is easy to learn. Also, its libraries and frameworks are more mature from a development point of view.

**Comment by AhmarZaidi** [comment]
I guess this issue can be closed now since we have already decided on Gol.
